lib/cb/version.rb

```diff
@@ -1,3 +1,3 @@
 module Cb
-  VERSION = '16.2.2'
+  VERSION = '17.0.0'
```

Comment:
Is this really a breaking change? It does now assume that `errors` is an array, but is that a deviation from the rest?